### PR TITLE
TableOfContents added in the README.md

### PR DESCRIPTION
**Pull request by ng29**
Hey @robbieaverill 

Added table of contents in the readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,20 @@ A high-fidelity fully-fledged eCommerce demo store built using the Commerce.js S
...
+# Table of Contents
+
+ * [Overview](#overview)
+ * [Prerequisites](#prerequisites)
+ * [Setup](#setup) 
+     * [Create a Chec account](#create-a-chec-account)
+ * [One-click Deploy with Netlify (recommended)](#one-click-deploy-with-netlify-recommended)
+ * [Manual setup and Netlify deployment](#manual-setup-and-netlify-deployment)
+ * [Setup using Chec CLI demo-store command](#setup-using-chec-cli-demo-store-command)
+ * [Caveats with data customization (IMPORTANT)](#caveats-with-data-customization-important)
```
Comment (robbieaverill) [inline README.md:23]:
```suggestion
   * [Caveats with data customization (IMPORTANT)](#caveats-with-data-customization-important)
```

Minor point, this is a subheading of the CLI based setup heading

**Review by robbieaverill — APPROVED** [review]
Nice, thanks!

**Comment by ng29** [comment]
> Nice, thanks!

Just one query that this PR will be eligible for Celebrate Hacktoberfest with Commerce.js!  ?? ?

**Comment by robbieaverill** [comment]
Yes, of course!

**Comment by ng29** [comment]
> Yes, of course!

Thanks alot @robbieaverill , cant wait to brag on twitter about commercejs SWAG.
HAPPY HACKTOBERFEST

Cheers

**Pull request merged**